optuna/integration/botorch.py

```diff
@@ -466,6 +488,10 @@ class BoTorchSampler(BaseSampler):
         n_startup_trials:
             Number of initial trials, that is the number of trials to resort to independent
             sampling.
+        consider_running_trials:
+            If True, the acquisition function takes into consideration the running parameters
+            whose evaluation has not completed. Enabling this option is considered to improve the
+            performance of parallel optimization.
```

Comment:
It would be great to show this argument is experimental in the docstring.

Reply:
I added a note section saying that this is experimental.